### PR TITLE
feat(sentry-apps): API to fetch/update SentryApp features

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
API to fetch all the Sentry App features and API to update a specific sentry app's feature category.

### CODE REVIEW

**Comment by ceorourke** [comment]
Can you add tests for these?

**Comment by MeredithAnya** [comment]
@NisanthanNanthakumar we haven't done this yet but I think we should technically be allowed to have multiple features for one integration, at least that was the intention with the `IntegrationFeature` model

File: src/sentry/api/endpoints/sentry_app_features.py [hunk 1]
```diff
@@ -14,3 +16,26 @@ def get(self, request, sentry_app):
             paginator_cls=OffsetPaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def put(self, request, sentry_app):
+
+        current = request.data.get("current")
+        value = request.data.get("value")
```
Comment (scefali) [inline src/sentry/api/endpoints/sentry_app_features.py:23]: @NisanthanNanthakumar why not use a serializer to do the validation logic?

File: src/sentry/api/endpoints/sentry_app_features.py [hunk 2]
```diff
@@ -14,3 +16,26 @@ def get(self, request, sentry_app):
...
+
+        features_dict = dict(Feature.as_choices())
+        if not current or not features_dict[current]:
+            return self.respond(
+                '"current": ["This field is required."]', status=status.HTTP_400_BAD_REQUEST
+            )
+
+        if not value or not features_dict[value]:
```
Comment (scefali) [inline src/sentry/api/endpoints/sentry_app_features.py:31]: @NisanthanNanthakumar shouldn't you do `features_dict.get(value)` to avoid a key error?

**Pull request closed without merging**